ci.yml

```diff
@@ -0,0 +1,12 @@
+script:
+        - cd build
+        - echo $PATH
+        - export HOME=$(dirname "$PWD")
```

Comment:
The CI system itself (your node.js code in this case) should be setting HOME to be the private workdir of the job rather than requiring every ci.yml to implement something like this.